### PR TITLE
configs for CF deployment

### PR DESCRIPTION
**Pull request by creed-victor**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-v12.22.3
+v14.17.0
```
Comment (swamp-thing-sovryn) [inline .nvmrc:1]: @creed-victor was any dependency breaking? I think I've seen that but I want to be sure the reason behind this change

File: next.config.js [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
+// const { version } = require('./package.json')
 
-const COMMIT_SHA =
-  process.env.COMMIT_SHA ||
-  process.env.NOW_GITHUB_COMMIT_SHA ||
-  execSync("git log --pretty=format:'%h' -n 1")
+// const COMMIT_SHA =
+//   process.env.COMMIT_SHA ||
+//   process.env.NOW_GITHUB_COMMIT_SHA ||
+//   execSync("git log --pretty=format:'%h' -n 1")
```
Comment (swamp-thing-sovryn) [inline next.config.js:12]: @creed-victor can we please remove the commented out lines? :pray:

File: .nvmrc (hunk 1 above)
Comment (creed-victor) [reply .nvmrc:1]: Yep, but can't remember which one

Comment (swamp-thing-sovryn) [reply .nvmrc:1]: no worries

**Review by swamp-thing-sovryn — APPROVED** [review]

**Pull request merged**